Feature/2019 07 08 fishbones for tomorrow (#519)
ordina-jworks/ordina-jworks.github.io: _posts/2019-07-08-Fishbones-for-tomorrow.md

---
layout: post
authors: [michaela_broeckx]
title: "Fishbones for Tomorrow"
image: /img/fishbones-for-tomorrow/main-image.jpg
tags: [Agile, AgileWorks, Agile Coaching, Liberating Structures, Root Cause Analysis, Powerful Techniques]
category: Agile
comments: true
---

For quite a few years now, I have been going steady with Japan, the birthplace of so many philosophies, teachings and practices preceding and gearing up to the Agile movement. 
This is how I came to discover the well-hidden connection between work and fish. 

> In Eastern Asia, the carp or koi is a deeply symbolic and highly respected animal, the presence and/or representation of which is considered a placeholder for luck, prosperity and a long life. 

Reviewer: ```suggestion
In Eastern Asia, the carp or koi is a deeply symbolic and highly respected animal, the presence and/or representation of it, is considered a placeholder for luck, prosperity and a long life. 
```

No surprise then, that the fish is sometimes used as a metaphor for good business practice and process improvement.

Well, here is another piece of Japanese fish for you to fry.

### Fish that were yesterday

> Without any doubt, Kaoru Ishikawa 石川馨 (1915-1989) was one of the game changer Japanese organizational theorists of our time. 

Reviewer: ```suggestion
Without any doubt, Kaoru Ishikawa 石川馨 (1915-1989) was one of the game changing Japanese organizational theorists of our time. 
```

With his near-contemporaries Noriaki Kano (of the famous Kano MVP and customer satisfaction model), W. Edwards Deming (of 14 points and PDCA cycle fame), and Hirotaka Takeuchi and Ikujiro Nonaka (who were the first to come up with SCRUM in ‘The New New Product Development Game’, 1986), Ishikawa sort of laid the foundations of organisational models as we know and use them today. 
Also, Ishikawa-sama is considered a key figure in the development of quality initiatives in Japan and elsewhere.

As a baseline, Ishikawa had in mind to change the way people think about life, including work. 

> His notion of organisation-wide quality control, cross-functional and cross-departmental, implied awareness of everybody always being one another’s supplier and customer, which in turn called for continued, multi-directional customer service. 

Reviewer: ```suggestion
His notion of organisation-wide quality control, cross-functional and cross-departmental, implied awareness of everybody always being each other's supplier and customer, which in turn called for continued, multi-directional customer service. 
```

This service would extend across the organisation itself, infecting not just the work and the output but even the everyday lives of those involved.

To this end, Ishikawa thought of a set of useful and relevant methods. 
One of these, and what Ishikawa is mostly famous for, is the fishbone diagram, a.k.a. the Ishikawa or cause-and-effect diagram, often used in the analysis of industrial processes.

Most commonly, Ishikawa diagrams are created to brainstorm over the multiple causes for a specific current event or defect. 
The current status, default status, or, perhaps in most cases, the defect, is shown as the fish's head, with the causes extending to the left as fishbones; the ribs branching off the backbone for major causes, with sub-branches for root-causes, up to as many levels as required, brainstormed or identified.
An example of a fishbone diagram is shown below: 

<img alt="Sushi" src="{{ '/img/fishbones-for-tomorrow/fishbone-example.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">

You could try doing fishbones at home with your children. 

> They are great fun to do – and they help your children understand that situations are mere results of a chain of previous events, and that nobody really is to blame for them.

Reviewer: ```suggestion
They are a lot of fun to do – and they help your children understand that situations are mere results of a chain of previous events, and that nobody really is to blame for them.
```

Fishbone structures with the head at the right side are beautiful – because they are in some way a reflection of evolution and natural selection. 
They can also be viewed as a representation of complexity, where multiple minute events, small circumstances and off-chance happenings, all contribute to ‘what is’, in the here and the now.

<img alt="Fishbones" src="{{ '/img/fishbones-for-tomorrow/fishbones.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">

### Turning around the arrow of time

Okay. 

> This is all pretty useful of course – but somehow, sometimes, in my work as an Agile Coach, I have been coming across teams more keen on exploring ahead and discovering further possibilities and opportunities, than keep looking backward. 

Reviewer: ```suggestion
This is all pretty useful of course – but somehow, sometimes, in my work as an Agile Coach, I have been coming across teams more keen on exploring ahead and discovering further possibilities and opportunities, than teams that keep looking backward. 
```

But once these areas are clear, every participant makes a deep dive into their personal knowledge and experience with regard to the goal and tries to gather as many ideas as possible, in silence.
Then, one by one, the team members present their ideas while others detect and formulate connections, doubles and insights.

This technique helps the team to orderly gather a myriad of ideas, and it really sets things in motion. 
Because from then onwards, with prioritisation and some concrete actions and owners, the fishbone forms the backbone for team improvement towards a self-defined goal.

### Collaboration is a whole new piece of fish

For us, Agilists and assorted kindred souls, the virtues of thinking and making stuff together are obvious. 
After all, there is hardly an approach in the Agile framework where co-creation is not written into the very DNA of who we are and what we do.

Does it need saying that being out in the world hunting for appropriate liberating structures permitting cooperation and collaboration has become a second nature for me?

It could be for you, too. 

> Think of this: that when turning around the arrow of time is permitted, if it facilitates working together - than everything else is permitted too, all the way up to imagining a world upside down.

Reviewer: ```suggestion
Think of this: when turning around the arrow of time is permitted, if it facilitates working together - then everything else is permitted too, all the way up to imagining a world upside down.
```
